### DISCUSSION TITLE
Swamped with feature request.

### DISCUSSION THREAD

**Discussion by JSuenram** [Ylianst/MeshCentral#3643] [Ideas]
Maybe we can create some sort of Roadmap from the dozens of feature requests.
Give some timelines when and if it will be worked on.

This would it make easier for users to watch when a certain feature will be ready.
Just an idea....

**Comment by Ylianst** [comment]
One idea I had was to encourage people to thumbs up the issues they want me to focus on. I can sort issues by thumbs up and so, it's a form of community voting. Of course, some issues are security related and so, always get high priority. I also give to priority to any contributions. If you fix language issues or make a pull request, that should come first since people put work into that and I don't want pull requests waiting too long.
